corehq/apps/reports/models.py

```diff
@@ -218,6 +219,18 @@ class ReportConfig(CachedCouchDocumentMixin, Document):
     end_date = DateProperty(default=None)
     datespan_slug = StringProperty(default=None)
 
+    @classmethod
+    def wrap(cls, data):
+        if 'filters' in data and ExpandedMobileWorkerFilter.slug in data['filters']:
+            data['filters'][LocationRestrictedMobileWorkerFilter.slug] = \
+                data['filters'][ExpandedMobileWorkerFilter.slug]
+            del data['filters'][ExpandedMobileWorkerFilter.slug]
+        try:
+            self = super(ReportConfig, cls).wrap(data)
+        except AttributeError:
```

Comment:
(Catching `AttributeError` because parent `wrap()` method may not be found by method resolution.)